example/sample_data/sample_table.csv

```diff
@@ -3,3 +3,4 @@ hive,gold,test_schema,test_table1,"1st test table","tag1,tag2,pii,high_quality",
 dynamo,gold,test_schema,test_table2,"2nd test table","high_quality,recommended",false,
 hive,gold,test_schema,test_view1,"1st test view","tag1",true,
 hive,gold,test_schema,test_table3,"3rd test","needs_documentation",false,
+hive,gold,test_schema,"test's_table4","4th test","needs_documentation",false,
```

Comment:
This is a test data from [here](https://github.com/amundsen-io/amundsen/issues/685)

Reply:
cc @instazackwu